[2022-06-03 19:36:22,394] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: nyc_bus_breakdown_pipeline.processing_data manual__2022-06-03T19:36:17.997157+00:00 [queued]>
[2022-06-03 19:36:22,404] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: nyc_bus_breakdown_pipeline.processing_data manual__2022-06-03T19:36:17.997157+00:00 [queued]>
[2022-06-03 19:36:22,405] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-03 19:36:22,405] {taskinstance.py:1358} INFO - Starting attempt 1 of 2
[2022-06-03 19:36:22,406] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-03 19:36:22,421] {taskinstance.py:1378} INFO - Executing <Task(PythonOperator): processing_data> on 2022-06-03 19:36:17.997157+00:00
[2022-06-03 19:36:22,426] {standard_task_runner.py:52} INFO - Started process 308 to run task
[2022-06-03 19:36:22,429] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'nyc_bus_breakdown_pipeline', 'processing_data', 'manual__2022-06-03T19:36:17.997157+00:00', '--job-id', '373', '--raw', '--subdir', 'DAGS_FOLDER/nycbus_etl.py', '--cfg-path', '/tmp/tmp16wlhh2v', '--error-file', '/tmp/tmpm8euqj8y']
[2022-06-03 19:36:22,430] {standard_task_runner.py:80} INFO - Job 373: Subtask processing_data
[2022-06-03 19:36:22,491] {task_command.py:370} INFO - Running <TaskInstance: nyc_bus_breakdown_pipeline.processing_data manual__2022-06-03T19:36:17.997157+00:00 [running]> on host 3b2e07cb0f3f
[2022-06-03 19:36:22,693] {taskinstance.py:1572} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=nyc_bus_breakdown_pipeline
AIRFLOW_CTX_TASK_ID=processing_data
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T19:36:17.997157+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-03T19:36:17.997157+00:00
[2022-06-03 19:36:23,782] {base.py:68} INFO - Using connection ID 's3_conn' for task execution.
[2022-06-03 19:36:23,784] {base_aws.py:210} INFO - Credentials retrieved from extra_config
[2022-06-03 19:36:25,997] {python.py:173} INFO - Done. Returned value was: None
[2022-06-03 19:36:26,013] {taskinstance.py:1401} INFO - Marking task as SUCCESS. dag_id=nyc_bus_breakdown_pipeline, task_id=processing_data, execution_date=20220603T193617, start_date=20220603T193622, end_date=20220603T193626
[2022-06-03 19:36:26,064] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-03 19:36:26,105] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
